src/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
     "version": "0.1",
     "requires": {
-        "cockpit": "137"
+        "cockpit": "231"
```

Comment:
strictly speaking, it is not really required. The page would work just fine with 230 for example. Even though I don't really think 137 is adequate version, but this one seem rather strict for no obvious reason.

Reply:
Hmm, good point, this really was for the tests only. That was a thinko then. Reverted.